pkg/steps/clusterinstall/template.go

```diff
@@ -143,6 +143,8 @@ objects:
         value: /tmp/artifacts
       - name: HOME
         value: /tmp/home
+      - name: XDG_RUNTIME_DIR
+        value: /tmp/home/run
```

Comment:
I don't see any `oc registry login` calls in either the `test` or `teardown` containers, are they done by `openshift-tests` / the installer?  Will those calls also need a `--to` argument?

Reply:
@bbguimaraes These variables are not needed at this time. I am adding this so we that do not have to add them later in case someone needs to call the `oc registry login`, or have other functionality that depends on this common directory. We may also decide that we want to change/remove the `.docker/config.json` preferences in the code. Although I am not sure how likely that is. I can remove the support for `XDG_RUNTIME_DIR` to slim down the PR if you prefer.

Reply:
ok, thanks. Next one to go is this one: https://github.com/openshift/release/pull/37726